WordPress/Classes/ViewRelated/Blog/SiteManagement/SettingsTitleSubtitleController.swift

```diff
@@ -0,0 +1,283 @@
+import UIKit
+import Gridicons
+
+
+/// Types the closures than can be provided as completion blocks
+typealias SettingsTitleSubtitleAction = ((SettingsTitleSubtitleController.Data) -> Void)
+
+
+/**
+ Presents a view controller with a textfiled and a textview, that can be used to create / edit a title and subtitle pair.
+*/
+final class SettingsTitleSubtitleController: UITableViewController {
+
+    /// The data to be presented on screen (i.e. title and subtitle).
+    final class Data {
```

Comment:
`Data` is a very generic name and there is already a foundation class named `Data`.  Maybe this could be named something more specific to this controller's use case?